shared/unattended/Fedora.ks

```diff
@@ -0,0 +1,48 @@
+install
+KVM_TEST_MEDIUM
+GRAPHICAL_OR_TEXT
+lang en_US
+keyboard us
+network --bootproto dhcp --hostname atest-guest
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+firstboot --disable
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
+zerombr
+poweroff
+KVM_TEST_LOGGING
+
+clearpart --all --initlabel
+autopart
+
+%packages --ignoremissing
+@standard
+@c-development
+@development-tools
+python
+net-tools
+sg3_utils
+%end
+
+%post
+# Output to all consoles defined in /proc/consoles, use "major:minor" as
+# device names are unreliable on some platforms
+# https://bugzilla.redhat.com/show_bug.cgi?id=1351968
+function ECHO { for TTY in `cat /proc/consoles | awk '{print $NF}'`; do source "/sys/dev/char/$TTY/uevent" && echo "$*" > /dev/$DEVNAME; done }
+ECHO "OS install is completed"
+grubby --remove-args="rhgb quiet" --update-kernel=$(grubby --default-kernel)
+dhclient
+chkconfig sshd on
+iptables -F
+systemctl mask tmp.mount
+echo 0 > /selinux/enforce
+sed -i "/^HWADDR/d" /etc/sysconfig/network-scripts/ifcfg-eth0
```

Comment:
Just curious, does anaconda add the HWADDR? I have recent-ish fedora VMs, both installed manually and imported images from virt-builder and none one of them has it in the network script, can you tell me what am I missing?

Reply:
Couple of years back it used to be like this. If you specified HWADDR and it found such device, it used it for this if-script. In case the HWADDR changed, it created another devices (eth2, ...) and kept eth0 disabled. Removing the HWADDR specification resulted in eth0 configuring card with any HWADDR, which is useful when you usually have just one nic. (by default eth2 would not be started and you'd end-up with no network connectivity after changing the HWADDR).

Reply:
But you go through libvirt and libvirt will add one, so it will always be the same, so we shouldn't need that bit, correct?

Reply:
also, if you have more that a single NIC, you should not be using the bios dev naming IOW use enp1s0 instead of eth0 etc. since I can see that could easily mess with the tests.

Reply:
When you use `--vt-type qemu` it will generate a fresh MAC address for each `vm.create()`.